build/Dockerfile.ci

```diff
@@ -119,7 +119,7 @@ RUN make build
 #├── windows_exporter.exe
 #└── wmcb.exe
 
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
+FROM registry.ci.openshift.org/openshift/release:golang-1.18
```

Comment:
@selansen  append "Follow-up to https://github.com/openshift/windows-machine-config-operator/commit/0bc812265d94fca75f0e745fe62bc0a9b4805525" to the commit message as a ref to the original commit. 

In addition, add `WINC-798` as a prefix to the PR title to link this PR to the Jira ticket.